addons/hr_presence/models/hr_employee.py

```diff
@@ -78,32 +76,21 @@ def _check_presence(self):
         for employee in all_employees:
             employee.hr_presence_state_display = employee.hr_presence_state
 
-    @api.model
-    def _action_open_presence_view(self):
-        # Compute the presence/absence for the employees on the same
-        # company than the HR/manager. Then opens the kanban view
-        # of the employees with an undefined presence/absence
-
-        _logger.info("Employees presence checked by: %s" % self.env.user.name)
-
-        self._check_presence()
-
-        return {
-            "type": "ir.actions.act_window",
-            "res_model": "hr.employee",
-            "views": [[self.env.ref('hr_presence.hr_employee_view_kanban').id, "kanban"], [False, "tree"], [False, "form"]],
-            'view_mode': 'kanban,tree,form',
-            "domain": [],
-            "name": _("Employee's Presence to Define"),
-            "search_view_id": [self.env.ref('hr_presence.hr_employee_view_presence_search').id, 'search'],
-            "context": {'search_default_group_hr_presence_state': 1,
-                        'searchpanel_default_hr_presence_state_display': 'to_define'},
-        }
+    def get_server_action_records(self, action_ids):
+        server_actions = self.env['ir.actions.server'].sudo().search([('value', '!=', False)])
+        return [{
+            'id': action.id,
+            'value': action.value,
+        } for action in server_actions]
```

Comment:
This public method returns in sudo the content of server actions :warning: :warning: :warning: